plugins/cue_points/base/lib/model/CuePointPeer.php

```diff
@@ -229,6 +229,12 @@ public static function validateMetadataObjects($profileField, $objectIds, &$erro
 	    return true;
 	}
 
+	public static function getEntry($objectId)
+	{
+		$cuePoint = self::retrieveByPKNoFilter($objectId);
```

Comment:
retrieveByPKNoFilter will return deleted object as well do you want this?
If not use retrieveByPK